module/Education/view/education/education/course.phtml

```diff
@@ -29,37 +41,57 @@ $this->headTitle($this->translate('Education'));
         </div>
         <hr>
         <div class="row">
-            <div class="col-md-8">
-                <h2><?= $this->translate('Exams and summaries') ?></h2>
-                <ul>
-                    <?php foreach ($course->getDocuments() as $document): ?>
-                        <li>
-                            <a href="<?= $this->url('education/course/download', [
-                                'code' => $course->getCode(),
-                                'id' => $document->getId()
-                            ]) ?>">
-                                <?php if ($document instanceof SummaryModel): ?>
-                                    <?php if (null !== ($author = $document->getauthor())): ?>
+            <div class="col-md-6">
+                <h2><?= $this->translate('Summaries') ?></h2>
+                <?php if (count($summaries) <= 0): ?>
+                    // phpcs:ignore Generic.Files.LineLength.TooLong -- user-visible strings should not be split
```

Comment:
This line is displayed on the actual page. Just moving it to a `<?php ?>` block won't help, Split the `translate()` below this and add the comment there:

```php
<?= $this->translate(
    // phpcs:ignore Generic.Files.LineLength.TooLong -- user-visible strings should not be split
    'Long text that is definity more than 120 characters',
) ?>
```